src/SPECS/hootenanny.spec.in

```diff
@@ -187,7 +187,11 @@ if [ "$1" = "1" ]; then
         RAND_PW=$(pwgen -s 16 1)
         sudo -u postgres createuser --superuser hoot || true
         sudo -u postgres psql -c "alter user hoot with password '$RAND_PW';"
-        sudo sed -i s/DB_PASSWORD=.*/DB_PASSWORD=$RAND_PW/ /var/lib/hootenanny/conf/DatabaseConfig.sh
+        if [ -f /var/lib/hootenanny/conf/DatabaseConfigDefault.sh ]; then
+            sudo sed -i s/DB_PASSWORD=.*/DB_PASSWORD=$RAND_PW/ /var/lib/hootenanny/conf/DatabaseConfigDefault.sh
+        else
+            sudo sed -i s/DB_PASSWORD=.*/DB_PASSWORD=$RAND_PW/ /var/lib/hootenanny/conf/DatabaseConfig.sh
```

Comment:
I think this is a no-op, as hoot versions bundled in this rpm will not have a password field in DatabaseConfig.sh any longer.